ui/pages/send/gas-display/gas-display.js

```diff
@@ -317,82 +321,83 @@ export default function GasDisplay({ gasError }) {
           ]}
         />
       </Box>
-      {(gasError || isInsufficientTokenError) && currentNetworkName && (
-        <Box
-          className="gas-display__warning-message"
-          data-testid="gas-warning-message"
-        >
+      {(gasError || isInsufficientTokenError || assetBalanceIsZero) &&
```

Comment:
Will including `assetBalanceIsZero` here cause the warning message to appear if the token balance is 0 on a token transfer?